chain-impl-mockchain/src/testing/arbitrary/transaction.rs

```diff
@@ -269,3 +270,58 @@ fn filter_accounts(x: &AddressDataValue) -> bool {
         _ => false,
     }
 }
+
+fn filter_utxo(x: &AddressDataValue) -> bool {
+    match x.address_data.kind() {
+        Kind::Single { .. } | Kind::Group { .. } => true,
```

Comment:
Why filter them?

Reply:
Because i want to calculate expected utxos only. For account there is another verifier.